### PR TITLE
Fix susceptance calculation

### PR DESCRIPTION
**Pull request by raphaelsaavedra**
From what we've seen, there are some bugs in these functions:

1. For `PhaseShiftingTransformer`, the function should return `imag(y_a)` instead of `1 /  imag(y_a)`, since `y_a` already is `1 / ...`.
2. There is a conversion to radians when `α` should already be in radians (#649) -- please confirm if this is really the case as I'm not 100% sure.
3. The first two functions are inconsistent with the third. Note that only the third one includes `im`, which means when we calculate `imag(y_a)` we actually get a minus sign in front of the expression due to the `im` in the denominator. A simple fix is to add the minus sign in the other functions for consistency.

A simple example of some of these problems:

```
julia> tr1
FBiqwdLBQB (PhaseShiftingTransformer):
   name: FBiqwdLBQB
   available: false
   active_power_flow: 0.0
   reactive_power_flow: 0.0
   arc: j28BGl4Bn7 -> 63tUEHHV7m: (Arc)
   r: 0.895
   x: 2.972
   primary_shunt: 0.0
   tap: 1.0
   α: 0.0
   rate: 6.69
   ext: Dict{String,Any}()
   forecasts: InfrastructureSystems.Forecasts: 0

julia> tr2 = TapTransformer(
       name=tr1.name, 
       available=tr1.available, 
       active_power_flow=tr1.active_power_flow, 
       reactive_power_flow=tr1.reactive_power_flow,
       arc=tr1.arc, 
       r=tr1.r, 
       x=tr1.x, 
       primary_shunt=tr1.primary_shunt, 
       tap=tr1.tap, 
       rate=tr1.rate
       ) # create identical tap transformer
FBiqwdLBQB (TapTransformer):
   name: FBiqwdLBQB
   available: false
   active_power_flow: 0.0
   reactive_power_flow: 0.0
   arc: j28BGl4Bn7 -> 63tUEHHV7m: (Arc)
   r: 0.895
   x: 2.972
   primary_shunt: 0.0
   tap: 1.0
   rate: 6.69
   services: 0-element Array{Service,1}
   ext: Dict{String,Any}()
   forecasts: InfrastructureSystems.Forecasts: 0

julia> PowerSystems.get_series_susceptance(tr1) # current version - wrong value
-3.2415238896366083

julia> get_series_susceptance(tr1) # fixed version (changed 1/imag to just imag)
-0.3084968780261265

julia> PowerSystems.get_series_susceptance(tr2) # different sign for tap transformer even though it is identical
0.3364737550471063
```

The small difference in absolute value is due to the resistance being considered only for `PhaseShiftingTransformer`.
This is also something to keep in mind - is it really the best approach to consider resistance for `PhaseShiftingTransformer` and disregard it for all other types of branch?

### CODE REVIEW

**Comment by jd-lara** [comment]
This pr looks good, I need to check that the conversion isn't there because we have bad parsing of the phase shifting data. We should merge this, but I need to confirm there isn't another issue with the conversion from matpower.

**Comment by raphaelsaavedra** [comment]
A nice little test to ensure these stay consistent would be to have several identical branches of different types, compute their susceptance and see if the results are identical (similar to what I did in the small example I wrote). This wouldn't work at the moment due to the difference in the resistance consideration, but if this becomes consistent across at least transformers it would be a simple test to add.

File: src/models/supplemental_accessors.jl [hunk 1]
```diff
@@ -14,12 +14,12 @@ _remove_aggregration_topology!(bus::Bus, ::Area) = bus.area = nothing
 """
 Calculate the admittance of AC branches
 """
-get_series_susceptance(b::ACBranch) = 1 / get_x(b)
-get_series_susceptance(b::TapTransformer) = 1 / (get_x(b) * get_tap(b))
+get_series_susceptance(b::ACBranch) = -1 / get_x(b)
+get_series_susceptance(b::TapTransformer) = -1 / (get_x(b) * get_tap(b))
```
Comment (jd-lara) [inline src/models/supplemental_accessors.jl:18]: This function should be 
```suggestion
function get_series_susceptance(b::TapTransformer)
    y = 1 / (get_r(b) + get_x(b) * 1im)
    y_a = y / (get_tap(b))
    return imag(y_a)
end
```

File: src/models/supplemental_accessors.jl [hunk 2]
```diff
@@ -14,12 +14,12 @@ _remove_aggregration_topology!(bus::Bus, ::Area) = bus.area = nothing
 """
 Calculate the admittance of AC branches
 """
-get_series_susceptance(b::ACBranch) = 1 / get_x(b)
-get_series_susceptance(b::TapTransformer) = 1 / (get_x(b) * get_tap(b))
+get_series_susceptance(b::ACBranch) = -1 / get_x(b)
```
Comment (jd-lara) [inline src/models/supplemental_accessors.jl:17]: same here 
```suggestion
get_series_susceptance(b::ACBranch) = imag(1 / (get_r(b) + get_x(b) * 1im))
```

**Review by jd-lara — COMMENTED** [review]
There seems to be a mismatch between the definition of susceptance in theory with the implementation in Matpower. See Line 62 of `makeBdc` function. 

There is also a mismatch in the calculation of the PTDF in the presence of phase shifters, refer also to `makeBdc` in matpower. I need to make more research into where the difference comes from.

File: src/models/supplemental_accessors.jl (hunk 2 above)
Comment (jd-lara) [reply src/models/supplemental_accessors.jl:17]: same here

**Comment by raphaelsaavedra** [comment]
Thanks, I guess this fixes the inconsistency in the consideration of resistance across those functions. I'll create a small test on this next.

**Comment by raphaelsaavedra** [comment]
Seems like some PTDF tests are failing after the latest changes.

**Comment by jd-lara** [comment]
> Seems like some PTDF tests are failing after the latest changes.

Yeah, that's because we are no longer following the same convention as Matpower.

**Pull request closed without merging**